bundles/ch.elexis.core/src/ch/elexis/core/constants/Elexis.java

```diff
@@ -5,30 +5,31 @@
 import java.net.URL;
 import java.util.Properties;
 
+import org.osgi.framework.FrameworkUtil;
 import org.slf4j.LoggerFactory;
 
 public final class Elexis {
-	
+
 	public static final String VERSION;
 	public static final String APPLICATION_NAME = "Elexis Core"; //$NON-NLS-1$
-	
+
 	static {
 		VERSION = initElexisBuildVersion();
 	}
-	
-	private static String initElexisBuildVersion(){
-		Properties prop = new Properties();
+
+	private static String initElexisBuildVersion() {
+		String version = FrameworkUtil.getBundle(Elexis.class).getVersion().toString();
 		String url_name = "platform:/plugin/ch.elexis.core.data/rsc/version.properties";
 		try (InputStream inputStream = new URL(url_name).openConnection().getInputStream()) {
 			if (inputStream != null) {
+				Properties prop = new Properties();
 				prop.load(inputStream);
-				return prop.getProperty("elexis.version").replace("-SNAPSHOT", "");
+				version = prop.getProperty("elexis.version");
```

Comment:
Bist du sicher dass du version komplett ersetzen willst? Nicht nur einfach qualifier replacement?

Reply:
Ja. Die Version via Datei version.properties wird von maven/tycho ermittelt und holt sich auch die gleiche Version. 
Wenn die Datei noch von einem alten Maven Build her existiert und man in der IDE die Version hochzieht, dann ist in diesem Fall alte Version und alter Timestamp drin.
Tritt bei mir nie auf, da ich dann immer sofort einen maven build erstelle.
Falls Du das trotzdem willst müsste ich den maven build anpassen und eher eine Property elexis.buildtimestamp anstelle von elexis.version erstellen.
Ist mir mehr Arbeit, aber wenn Du meinst, dass dies besser ist kann ich es schon machen.